gradle.properties

```diff
@@ -16,3 +16,22 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+
+
+GROUP=co.paystack.android.design.widget
+VERSION_NAME=1.0.4
+
+POM_DESCRIPTION=Pinpad for Paystack Android SDK
+
+POM_URL=https://github.com/PaystackHQ/android-pinpad
+POM_SCM_URL=https://github.com/PaystackHQ/android-pinpad
+POM_SCM_CONNECTION=scm:git:github.com/PaystackHQ/android-pinpad.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://github.com/PaystackHQ/android-pinpad.git
+
+POM_LICENCE_NAME=The Apache Software License, Version 2.0
+POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=paystack
+POM_DEVELOPER_NAME=Paystack
+POM_DEVELOPER_EMAIL=developers@paystack.co
```

Comment:
Is this a typo? Should it be '.com' ?

Reply:
Nah. It's correct. Paystack's first domain was a .co